packages/generator/templates/operations.ts.hbs

```diff
@@ -1,153 +1,156 @@
 {{#each model.encodes.endPoints}}
```

Comment:
Hard to compare the changes with tabs removed..

Reply:
I totally agree, please revert whitespace changes.

NOTE: @kalagarraj if you click the gear at the top while reviewing the file changes you can select an option that says something 'hide whitespace changes"

I recommend reverting them, but this at least allows you to review it in the time being.

Reply:
Again, this is WIP, this will be resolved prior to merge. As noted in my earlier comment, the whitespace changes can be easily ignored for now :)